docker-compose.yml

```diff
@@ -35,6 +35,8 @@ services:
       ACCEPT_EULA: "Y"
       SA_PASSWORD: MyS3cr3tPassw0rd
 
+  # Setup the runner
+  # we need to set the localhost and root password as they can be different
   # Setup Lighthouse
   lighthouse:
     image: lighthouse:develop
```

Comment:
Shouldn't this be `build: .` instead of using `image`?

Reply:
We had some problem in local, it was not rebuilding the docker image on docker compose up. That was a bugfix added in develop.